src/main/server/services/helperProcess/index.ts

```diff
@@ -34,8 +34,7 @@ export class BlueBubblesHelperService {
         const pApiEnabled = Server().repo.getConfig("enable_private_api") as boolean;
         if (!pApiEnabled) return;
 
-        // eslint-disable-next-line no-nested-ternary
-        const macVer = isMinMonteray ? "macos12" : isMinBigSur ? "macos11" : "macos10";
+        const macVer = isMinBigSur ? "macos11+" : "macos10";
```

Comment:
I'd like to keep this just as macos11 (without the +)

Reply:
Would you like me to just rename the bundle back to macos11, or should I also revert the change to allow the macos11 bundle to be installed on Monterey?